gptel-context.el

```diff
@@ -163,32 +173,59 @@ context chunk.  This is accessible as, for example:
         (eq buffer-file-coding-system 'no-conversion))
     (file-missing (message "File \"%s\" is not readable." path))))
 
+(defun gptel-context--add-text-file (path)
```

Comment:
We now have `--add-text-file`, `--handle-binary` and `--handle-directory`.  Can we unify the naming here to `--handle-text-file`, `--handle-binary-file` and `--handle-directory`?

I actually prefer `--add-text-file`, `--add-binary-file` and `--add-directory`.  It's okay that `--add-directory` both adds and removes directories, because `gptel-add` both adds and removes context.